### DISCUSSION TITLE
怎么加强网络搜索结果的展示呢？

### DISCUSSION THREAD

**Discussion by elarbor** [vuejs/vitepress#3674] [General]
vitepress怎么配置seo呢？在百度、必应、谷歌等所搜能展示我的vitepress项目比较靠前

**Comment by zeMinng** [comment]
花钱呗，没有什么钱解决不了的

**Reply by elarbor** [reply]
vitepress不能配置每个文档页面的keywords和description吗？

**Reply by zeMinng** [reply]
你觉得会有用吗。你可以尝试hhh，没有那么简单的！
